test/saml2.coffee

```diff
@@ -104,13 +104,14 @@ describe 'saml2', ->
 
     describe 'check_saml_signature', ->
       it 'accepts signed xml', ->
-        assert saml2.check_saml_signature(get_test_file("good_assertion.xml"), get_test_file("test.crt"))
+        result = saml2.check_saml_signature(get_test_file("good_assertion.xml"), get_test_file("test.crt"))
+        assert.deepEqual result, [get_test_file("good_assertion.xml")]
 
       it 'rejects xml without a signature', ->
-        assert.equal false, saml2.check_saml_signature(get_test_file("unsigned_assertion.xml"), get_test_file("test.crt"))
+        assert.equal null, saml2.check_saml_signature(get_test_file("unsigned_assertion.xml"), get_test_file("test.crt"))
 
       it 'rejects xml with an invalid signature', ->
```

Comment:
Worth adding a test that would fail with the previous code, and pass with this one, to help prevent regressions?